### PR TITLE
Fix issue with Text ant Textarea not resizing

### PR DESCRIPTION
**Pull request by SylvainCorlay**
@jdfreder @ellisonbg .

### CODE REVIEW

**Comment by ellisonbg** [comment]
Looks good, merging.

File: ipywidgets/static/widgets/js/widget_string.js [hunk 1]
```diff
@@ -128,11 +128,7 @@ define([
             /**
              * Set a css attr of the widget view.
              */
-            if (name == 'padding' || name == 'margin') {
-                this.$el.css(name, value);
-            } else {
-                this.$textbox.css(name, value);
-            }
+            this.$el.css(name, value);
```
Comment (jdfreder) [inline ipywidgets/static/widgets/js/widget_string.js]: I think the base class defines this method, exactly like this.  No need to redefine it here - go ahead and delete the whole method.

**Comment by jdfreder** [comment]
Whoops, looks like that was merged before my comment was seen.

**Comment by SylvainCorlay** [comment]
opening another one.

**Pull request merged**